src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -64,6 +67,7 @@ define(function (require, exports, module) {
         this._color = tinycolor(color);
         this._originalColor = color;
         this._redoColor = null;
+        this._isUpperCase = PreferencesManager.get("uppercaseColors");
```

Comment:
@le717 As a workaround, I think that line of `testConvert()` could be changed to:

```js
    expect(colorEditor.getColor().getOriginalInput()).toBe(result);
```